lib/tf_lock/tf_lock_acquire.py

```diff
@@ -46,7 +48,7 @@ def tf_lock_acquire(root_module: Path) -> ExitCode:
 
             tf_lock_user = TFLockUser.from_string(lock_user)
             if tf_lock_user.pr_url:  # a pr holds the lock.
-                if is_pr_closed(tf_lock_user.pr_url):
+                if PR(URL(tf_lock_user.pr_url)).is_closed():
```

Comment:
p3 would be good for `pr_url` to already have type URL.

Reply:
i wonder if the URL type is actually needed... it doesn't seem to do much